plexupdate.sh

```diff
@@ -658,9 +664,11 @@ if [ "${AUTOINSTALL}" = "yes" ]; then
 	fi
 
 	${DISTRO_INSTALL} "${DOWNLOADDIR}/${FILENAME}"
-	if [ $? -ne 0 ]; then
+	RET=$?
+	if [ ${RET} -ne 0 ]; then
 		# Clarify why this failed, so user won't be left in the dark
 		error "Was unable to install due to problems with package from plex.tv or your local linux setup"
```

Comment:
This should provide the command that caused the error for easier debugging.

e.g. `error "Failed to install update. The command '${DISTRO_INSTALL} "${DOWNLOADDIR}/${FILENAME}"' returned error code ${RET}"`

I think if people see apt/dnf/yum in the error they'll know that it wasn't a problem with plexupdate itself. And if they copy and paste the error into an issue we'll get to skip one round of back and forth since we can get a lot of useful info from the command line that was used for the install.